### DISCUSSION TITLE
Email Configuration Error

### DISCUSSION THREAD

**Discussion by Mari-tech2000** [horilla-opensource/horilla#253] [General]
Tried configuring Mail Server with Gmail as below
![image](https://github.com/user-attachments/assets/aea28e18-6e5e-4189-8c0e-62f5fe0f49a7)

but getting this error, what I'm supposed to do.
![image](https://github.com/user-attachments/assets/20e31841-a321-4b77-bc27-ebeed255e42d)

created a folder staticfiles and created files as mentioned but now I'm getting another error
![image](https://github.com/user-attachments/assets/9374b9f0-7030-4edd-a533-0092c2ea72dd)
just changed port number from 587 to  465 and it's now working
Regards,

**Comment by VyvaHart** [comment]
Hi @Mari-tech2000 

Check Port and SSL/TLS Settings one more time
if you using SSL protocol you should set port 465

if it doesn't work, check **email host** (it should be: **smtp.gmail.com**)

and check one more time .png path. On screenshot i see another slashes ('/') after "images"

Good Luck!

**Reply by VyvaHart** [reply]
The solution may be to change protocol from SSL to TLS and set port 587 as well

**Reply by Mari-tech2000** [reply]
Thank you the error has been sorted
